### ISSUE TITLE
Error: ER_PARSE_ERROR: You have an error in your SQL syntax "SELEC"

### ISSUE THREAD

**Issue by davidportella** [balderdashy/sails-mysql#225]
Hi, I present this problem when you try to add an existing table in mysql my sails project

```
Details:  Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SELEC FROM `cellphone_status` AS `cellphone_status`' at line 1
```

```
// Model in Pos.js
...
module.exports = {
    connection   : 'mysqlDevel',
    tableName    : 'cellphone_status',
    migrate      : 'alter',
    schema       : true,
    autoCreatedAt: false,
    autoUpdatedAt: false,
    autoPK       : false,
...
```

```
// Service in PosService.js
...
Pos
    .find(where)
    .then(function (data) {
         console.log("Testing service");
         console.log(data);
         return resolve(data);
     })
    .catch(function (e) {
          sails.log(e);
    })
   ;
...
```

google searching I found

https://groups.google.com/forum/#!topic/sailsjs/D1u-R4cDwxA

and

http://stackoverflow.com/questions/29067583/why-cant-i-name-my-sailsjs-models-the-same-as-my-mysql-tables

but no solutions, so I see the problem is attributed to utils.js by query.slice by eliminating the T could be SELECT and SELECT leaving

https://github.com/balderdashy/sails-mysql/blob/master/lib/utils.js

**Comment by sailsbot** [comment]
Thanks for posting, @davidportella.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
